src/Taboo/TabooRoom.py

```diff
@@ -40,15 +50,110 @@ def postProcessConnect(self, ws):
         """Invoked when a new client (websocket) connects from the room.
         Note that no messages have been exchanged yet
         """
+        self.playerByWs[ws] = None
         self.publishGiStatus()
 
     def postProcessDisconnect(self, ws):
         """Invoked when a client disconnects from the room"""
+        if self.playerByWs.get(ws, None):
```

Comment:
why not:

   assert ws in self.playerByWs
   player = self.playerByWs[ws]
   if player:
      # additional logic
   del self.playerByWs[ws]

(avoiding try..catching)

Reply:
Changed as suggested